.github/CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+#CODEOWNERS file
+
```

Comment:
We could add a link to https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners and briefly explain the purpose of this file.